sql/core/pom.xml

```diff
@@ -130,6 +130,12 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.apache.avro</groupId>
+      <artifactId>avro</artifactId>
+      <version>1.8.0</version>
```

Comment:
Thank you for review again.

> should this not just use the same version of Avro as the rest of the project? avro.version is already defined to be 1.7.7 in the parent POM. I'd think we need to use that instead.

1. Do you mean to upgrade `avro` from 1.7.7 into 1.8.0 for the whole Spark project at this time?

> Second, is this really just a test issue?

2. While `sql/hive` depends on avro 1.7.7 API and 1.8.2 does not seem to break it. However, [ParquetAvroCompatibilitySuite](https://github.com/apache/spark/blob/master/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetAvroCompatibilitySuite.scala#L46) uses `AvroParquetWriter` which depends on 1.8.2 only.

> Therefore, I wonder if it is best to simply remove the version and scope here. That may be the most correct thing?

3. If we remove the version, the avro means 1.7.7. It will not resolve the maven build failure.